test/unit/component/registry/bpmn-model-filters.test.ts

```diff
@@ -518,4 +518,47 @@ describe('Bpmn Model filters', () => {
       bpmnElementKind: ShapeBpmnElementKind.POOL,
     });
   });
+
+  it('Filter a black pool (No processRef)', () => {
```

Comment:
```suggestion
  it('Filter a black pool', () => {
```

Reply:
For someone which doesn't know each term of the BPMN spec, I think it's better to precise what is a Black Pool.
Even me, I don't remember when I write the test ^^